tests/foreman/ui/test_rhc.py

```diff
@@ -0,0 +1,166 @@
+        logger.debug(f"Activation key: {ak} \n Organization: {module_rhc_org}")
+
+
+@pytest.mark.tier3
+def test_positive_configure_cloud_connector(
+    session,
+    module_target_sat,
+    fixture_enable_rhc_repos,
+    module_rhc_org,
+    fixture_setup_rhc_satellite,
+):
+    """Install Cloud Connector through WebUI button
+
+    :id: 67e45cfe-31bb-51a8-b88f-27918c68f32e
+
+    :Steps:
+
+        1. Navigate to Configure > Inventory Upload
+        2. Click Configure Cloud Connector
+        3. Open the started job and wait until it is finished
+
+    :expectedresults: The Cloud Connector has been installed and the service is running
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Critical
+
+    :BZ: 1818076
+    """
+    # Copy foreman-proxy user's key to root@localhost user's authorized_keys
+    module_target_sat.add_rex_key(satellite=module_target_sat)
+
+    # Set Host parameter source_display_name to something random.
+    # To avoid 'name has already been taken' error when run multiple times
+    # on a machine with the same hostname.
+    host = module_target_sat.api.Host().search(query={'search': module_target_sat.hostname})[0]
+    parameters = [{'name': 'source_display_name', 'value': gen_string('alpha')}]
+    host.host_parameters_attributes = parameters
+    host.update(['host_parameters_attributes'])
+
+    with session:
+        session.organization.select(org_name=module_rhc_org.name)
+        if session.cloudinventory.is_cloud_connector_configured():
+            pytest.skip(
+                'Cloud Connector has already been configured on this system. '
+                'It is possible to reconfigure it but then the test would not really '
+                'check if everything is correctly configured from scratch. Skipping.'
+            )
+        session.cloudinventory.configure_cloud_connector()
+
+    template_name = 'Configure Cloud Connector'
+    invocation_id = (
+        module_target_sat.api.JobInvocation()
+        .search(query={'search': f'description="{template_name}"'})[0]
+        .id
+    )
+    wait_for(
+        lambda: module_target_sat.api.JobInvocation(id=invocation_id).read().status_label
+        in ["succeeded", "failed"],
+        timeout="1500s",
+    )
+
+    job_output = module_target_sat.cli.JobInvocation.get_output(
+        {'id': invocation_id, 'host': module_target_sat.hostname}
+    )
+    logger.debug(f"Invocation output>>\n{job_output}\n<<End of invocation output")
+    # if installation fails, it's often due to missing rhscl repo -> print enabled repos
+    repolist = module_target_sat.execute('yum repolist')
+    logger.debug(f"Repolist>>\n{repolist}\n<<End of repolist")
+    # print rhc status
+    rhc_status = module_target_sat.execute('rhc status')
+    logger.debug(f"rhc status>>\n{rhc_status}\n<<End of rhc status")
+    # print rhcd log
+    rhcd_log = module_target_sat.execute('journalctl --unit=rhcd')
+    logger.debug(f"journalctl log>>\n{rhcd_log}\n<<End of log")
```

Comment:
Is such logging necessary? 

If I do:
```python
from robottelo.hosts import Satellite
from robottelo.logging import logger
sat = Satellite('dhcp-3-150.ourlab.com')
logger.debug(sat.execute('cat /etc/resolv.conf'))
```

I see the stdout being duplicated in `robottelo.log`:
```log
[D 220727 18:20:48 hosts:108] dhcp-3-150.ourlab.com command result:
    # Generated by NetworkManager
    search ourlab.com
    nameserver 10.2.32.1
    nameserver 10.11.5.19
    
2022-07-27 18:20:48 - robottelo - DEBUG - # Generated by NetworkManager
    search ourlab.com
    nameserver 10.2.32.1
    nameserver 10.11.5.19
```

Reply:
ah, nice. We don't need it then.